scripts/integration-tests

```diff
@@ -64,6 +65,10 @@ kubectl create -f conf/sample/networkservice-channel.yaml
 kubectl create -f conf/sample/networkservice-endpoint.yaml
 kubectl create -f conf/sample/networkservice.yaml
 kubectl get pod,crd,NetworkService,NetworkServiceEndpoint,NetworkServiceChannel --all-namespaces
+
+# Need to get kubeconfig full path
+k8sconfig=$HOME'/.kube/config'
+go test ./plugins/crd/... -v --kube-config=$k8sconfig
```

Comment:
And then subsequently update this to:

`go test ./plugins/crd/... -v --kube-config=$K8SCONFIG`